library/std/src/path.rs

```diff
@@ -1832,8 +1776,8 @@ impl Path {
     /// Any non-Unicode sequences are replaced with
     /// [`U+FFFD REPLACEMENT CHARACTER`][U+FFFD].
     ///
-    /// [`Cow<str>`]: ../borrow/enum.Cow.html
-    /// [U+FFFD]: ../char/constant.REPLACEMENT_CHARACTER.html
+    /// [`Cow<str>`]: Cow
+    /// [U+FFFD]: super::char::REPLACEMENT_CHARACTER
```

Comment:
nit: I think this would be more clear as
```suggestion
    /// [U+FFFD]: crate::char::REPLACEMENT_CHARACTER
```